### ISSUE TITLE
Using editor with DnD extension hinders mouse and keyboard interaction 

### ISSUE THREAD

**Issue by apijay** [SitePen/dgrid#1020]
I was trying to use editor and DnD in my Dgrid and was facing some issues. So i tried to reproduce the issue in Dgrid Test pages and was able to do successfully.
I made slight changes to Dgrid Test page(dgrid/test/extensions) "DnD.html" to use editor for creating columns. The aim was to use editor to change text of column "name".

Before(line 87 in DnD.html):
columns: [
          {label:"Name", field:"name", sortable: false},
          {label:"Description", field:"description", sortable: false}
],

After(line 87 in DnD.html):
columns: [
           editor({label:"Name", field:"name", sortable: false},"text", "click"),
           {label:"Description", field:"description", sortable: false}
],

Now on making above changes, I am facing below issues on clicking the column "name": 
--  Mouse can't be used to position the cursor.
-- 'Ctrl-A' to select all text in a field doesn't work.

Below is the link to issue:
http://apiworks.us/dgridIssues/dgrid/test/extensions/DnD.html

To sum up, this issue only occurs when editor is used with DnD . So on line 55 in DnD.html if you use var DnDGrid = declare([Grid, Selection, Keyboard]); instead of var DnDGrid = declare([Grid, Selection, DnD, Keyboard]); , the editor would work totally fine.

**Comment by MicahStevens** [comment]
Was this fixed? I still see this problem in my code. Looks like you just closed it.

**Comment by MicahStevens** [comment]
I'm seeing this as well, DnD provides the skipForm parameter, but this doesn't seem to work, or isn't passed to DnD via the extension. I'm still troubleshooting. I think this would be the solution, or should be. I'll try to make it work and open a pull request.
